lambdas/process_webhooks/process_webhooks.py

```diff
@@ -171,9 +198,34 @@ def lambda_handler(event, _context):
         for url in urls:
             results.append(_send_message(url, payload, headers))
 
-    results_count = _process_results_for_failures(results)
-
-    if results_count.get('failure'):
-        raise StandardError(results_count)
-
-    return results_count
+        results_count = _process_results_for_failures(results)
+
+        if results_count.get('failure'):
+            if not from_queue:
+                # The transmission was a failure, if its not
+                # already in the queue, add it.
+                queue_name = _get_target_queue()
+                response = _send_to_queue(event, queue_name)
+            raise StandardError(results_count)
+
+        return results_count
+    elif spigot_state == "OFF":
+        # Since the spigot is off, send the event
+        # to SQS for future processing. However,
+        # if the message is already in the queue do
+        # nothing.
+        if from_queue:
+            raise StandardError(
+                "The spigot is OFF. No messages should be "
+                "sent from the queue."
+            )
+        else:
+            queue_name = _get_target_queue()
+            response = _send_to_queue(event, queue_name)
+            return response
```

Comment:
nit: depending on the spigot_state, this method on success will either return results_count or response, which are two different types of object. Which is a bit odd. Might be better to decide what if anything it really should return. I'm guessing the answer would be based on testability. 
If we had enforced a standard in this repo of using a certain docstring style that included parameters, return value, what it raises, etc. then we would force ourselves to consider stuff like that ahead of time. We should probably do that.